docs/cimpl6LanguageReference.md

```diff
@@ -705,11 +693,13 @@ includes TumorDepth 0..1
 <!-- Carmela - figure out what happened to the <datatype> info -->
 | In the class... | Use the grammar... |
 |----------|---------|
-| `Element` | `Value only` <datatype>  |
-| `Element` | `Value only` <datatype1> or <datatype2> or <datatype3> etc.  |
-| `Entry` or `Group` | `<Property> only <datatype>` <br> >**Note**: this is allowed only for a single data type; to constrain to multiple choices, use the `substitute` constraint (see Example 2, below)|
+| `Element` | `Value only` datatype  |
```

Comment:
The text datatype, datatype2, etc. is substitution text and should be italicized

Reply:
Fixed. 
| In the class... | Use the grammar... |
|----------|---------|
| `Element` | <code>Value only <i>datatype</i></code>  |
| `Element` | <code>Value only <i>datatype1</i> or <i>datatype2</i> or <i>datatype3</i></code> etc.  |
| `Entry` or `Group` | <code><i>Property</i> only <i>datatype</i></code> |